docs/git-integration.md

```diff
@@ -0,0 +1,34 @@
+# Git Integration for DevOps & GitOps
+
+Git is integrated into Monokle to tackle GitOps seamlessly within the application. If you work within a DevOps culture our Git integration provides advantages for you and your team by keeping a history of changes made to your applications stored in Git to ensure access to a reliable audit trail of your changes.
+
+## **Getting Started with a New Project: Clone a Git repository**
+
+You can use a Git repository URL to start a new project. The repository will be cloned to the selected location and then opened into Monokle.
```

Comment:
You can start a new clone form a Git repository URL. The repo will be cloned locally and then opened into Monokle

Reply:
Changed.